README.md

```diff
@@ -420,9 +420,9 @@ ANSWERS.addComponent('SearchBar', {
   promptHeader: 'Header',
   // Optional, no default
   placeholderText: 'Start typing...',
-  // Optional, defaults to false
-  autoFocus: true,
-  // Optional, when auto focus on load,  open the autocomplete
+  // Optional, auto focuses the search bar. Defaults to false
+  autoFocus: false,
+  // Optional, opens the autocomplete suggestions on page load. Defaults to false. Requires autoFocus to be set to true
   autoCompleteOnLoad: false,
```

Comment:
Should this be called `autocompleteOnLoad`?

Reply:
ya, that's the whole purpose of this PR. I don't understand how I missed that change 😂